daemon/containerd/image_builder.go

```diff
@@ -332,32 +332,19 @@ func (rw *rwlayer) Commit() (_ builder.ROLayer, outErr error) {
 	}()
 
 	// Temporarily unmount the layer, required by the containerd windows snapshotter.
```

Comment:
This is no longer temporary. We also might want to mention that the windowsfilter graphdriver also remounts it afterwards, as it justifies the new final comment paragraph.

Reply:
Whoops, missed that part of the comment. Fixed it.

Reply:
> windowsfilter graphdriver also remounts it afterwards

Hmmm are you sure about that? I looked into the code and couldn't find it.

Reply:
In fact, there's this code in the image service (so level above the graphdriver):
https://github.com/moby/moby/blob/83de55b37040f21034f198d8e8afc763b78d4d6e/daemon/images/image_commit.go#L85-L93

suggesting the opposite - that the Diff actually needs the layer to be mounted.
OTOH, this PR works so it looks like that comment might actually be outdated/wrong?